src/app/ApplicationTemplate.Shared.Views/Startup.cs

```diff
@@ -39,6 +40,8 @@ protected override async Task StartViewServices(IServiceProvider services, bool
 				await AddSystemBackButtonSource(services);
 
 				HandleSystemBackVisibility(services);
+
+				ValidatorOptions.LanguageManager = new FluentValidationLanguageManager();
```

Comment:
Did you try to do this earlier? like in the initialization phase or just after the initialization phase?

Reply:
I haven't. Would be it be better to do so earlier?

Reply:
There's a potential race condition currently because you do the initial navigation before setting this. I would put it in the "OnInitialized" method if possible.

Reply:
done